packages/core/admin/admin/src/content-manager/components/InputJSON/Label.js

```diff
@@ -1,18 +1,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
 import { useIntl } from 'react-intl';
 import { FieldLabel } from '@strapi/design-system/Field';
-import { Box } from '@strapi/design-system/Box';
-import { Flex } from '@strapi/design-system/Flex';
 
-const LabelAction = styled(Box)`
```

Comment:
I found that json field type couldn't be localised, do we really need this `labelAction` for any other reason? 👀 @gu-stav

Reply:
`labelAction` is used to pass the is-localized indicator around in the CM (the little globe next to the label). I think it can be passed to `FieldLabel` directly using `<FieldLabel action={labelAction} />` if I am not mistaken?

Reply:
I don't know why the JSON field would not be localizable. I'd say that is a bug and we should continue to pass `action` around and rather see why the option is missing from the field setup.

Wdyt?

Reply:
sounds good to me, will add a ticket in the backlog to figure this out and revert my changes there!

Reply:
Maybe you can add it to GitHub too? I believe this is a good issue to be picked up by the community or gitstart :)

Reply:
Btw. I think you can keep the bulk of you changes - to move `labelAction` into the `action` prop is a much better approach than what we had before.

Reply:
actually my bad - I tested on a non localized CT 🤦‍♀️
json field type can be localized all good 🙇‍♀️

<img width="1356" alt="image" src="https://user-images.githubusercontent.com/71838159/208459060-ec0a8a4e-0986-441b-9030-02736d02efcc.png">
<img width="557" alt="image" src="https://user-images.githubusercontent.com/71838159/208459303-57b1af32-2ca9-456c-8101-bf76217c3495.png">